lib/query.js

```diff
@@ -20,6 +20,10 @@ export const isPos = node => _isOp('pos', node)
 export const isAbs = node => _isOp('abs', node)
 export const isFact = node => _isOp('fact', node)
 export const isNthRoot = node => _isOp('nthRoot', node)
+export const isFraction = (node) => query.isNeg(node)
+    ? query.isDiv(node.args[0])
+    : query.isDiv(node)
```

Comment:
This could be a bit more robust so that it handles things like `1/2 - -2/3`. 
```
query.isNeg(node) ? query.isFraction(node.args[0]) : query.isDiv(node)
```